core/src/test/java/nl/nn/adapterframework/jdbc/JdbcTestBase.java

```diff
@@ -58,24 +61,25 @@
 	@Parameterized.Parameter(0)
 	public @Getter TransactionManagerType transactionManagerType;
 	@Parameterized.Parameter(1)
-	public DataSource dataSource;
+	public String productKey;
+	private DataSource dataSource;
 
-	private @Getter DbmsSupportFactory dbmsSupportFactory = new DbmsSupportFactory();
+	private @Getter IDbmsSupportFactory dbmsSupportFactory;
 	protected @Getter IDbmsSupport dbmsSupport;
 
 	@Parameters(name= "{0}: {1}")
 	public static Collection data() throws NamingException {
 		TransactionManagerType type = TransactionManagerType.DATASOURCE;
-		List<DataSource> datasources;
+		List<String> datasources;
```

Comment:
Type is van DataSource naar String gegaan. Zou je de naam ook aan kunnen passen? Weet nu niet of het 'datasourceNames' of 'productKeys' zijn.